tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -446,7 +447,7 @@ def __generator(data, target, batch_size):
     yield from zip(data_batches, target_batches)
 
 
-class __GeneratorClass:
+class __GeneratorClass(GeneratorType):
```

Comment:
Keras 3 also validates the input type, if this doens't inherit `GeneratorType`, Error occurs:
```
ValueError: Unrecognized data type: x=<test_tensorflow2_autolog.__GeneratorClass object at 0x7feaff0a96a0> (of type <class 'test_tensorflow2_autolog.__GeneratorClass'>)
```